apps/nextjs/src/app/dashboard/organizations/[org_slug]/create/create-tournament.tsx

```
@@ -0,0 +1,513 @@
+            </div>
+            {formData.playerCap && (
+              <div>
+                <Label htmlFor="maxPlayers">Max Players</Label>
+                <Input
+                  name="maxPlayers"
+                  type="number"
+                  value={formData.maxPlayers}
+                  onChange={(e) =>
+                    setFormData({
+                      ...formData,
+                      maxPlayers: parseInt(e.target.value, 10),
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)